packages/app/src/sections/homepage/ShortList.tsx

```diff
@@ -17,35 +16,75 @@ import { Body } from 'components/Text'
 import ROUTES from 'constants/routes'
 import useENS from 'hooks/useENS'
 import useGetFuturesCumulativeStats from 'queries/futures/useGetFuturesCumulativeStats'
-import useGetStats from 'queries/futures/useGetStats'
 import { StackSection } from 'sections/homepage/section'
 import { Title } from 'sections/homepage/text'
+import { fetchFuturesStats } from 'state/home/actions'
+import { useAppDispatch, useAppSelector } from 'state/hooks'
+import { FetchStatus } from 'state/types'
 import { SmallGoldenHeader, WhiteHeader } from 'styles/common'
 import media from 'styles/media'
 
+type TraderENSProps = {
+	trader: string
+	traderShort: string
+}
+
+const TraderENS: FC<TraderENSProps> = ({ trader, traderShort }) => {
```

Comment:
Could be good to wrap this in memo and I might be better to move to it's own file for re-use

Reply:
Done.